src/main/java/appeng/client/Hotkeys.java

```diff
@@ -0,0 +1,32 @@
+package appeng.client;
+
+import java.util.HashMap;
+
+import org.lwjgl.glfw.GLFW;
+
+import net.fabricmc.fabric.api.client.keybinding.v1.KeyBindingHelper;
+import net.minecraft.client.KeyMapping;
+
+import appeng.hotkeys.LocatingServicesImpl;
+
+public class Hotkeys {
+
+    private static final HashMap<String, Hotkey> HOTKEYS = new HashMap<>();
+
+    static {
+        LocatingServicesImpl.REGISTRY.forEach((id, locatingServices) -> registerHotkey(createHotkey(id)));
```

Comment:
I don't like this since it is dependent on mods registering their stuff before this class is loaded. Maybe this class could be combined with the hotkey action registry impl, and create the mappings as they are registered.